sdks/java/testing/load-tests/src/main/java/org/apache/beam/sdk/loadtests/SyntheticDataPubSubPublisher.java

```diff
@@ -73,6 +81,12 @@
     String getInsertionPipelineTopic();
 
     void setInsertionPipelineTopic(String topic);
+
+    @Description("Kafka server address")
+    @Default.String("")
```

Comment:
Could you remove this default and do a null-check instead?

Reply:
one last thing: we need to change the class name as it's not only about `PubSub` now :)